packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -566,6 +572,63 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     }).then<int>((_) => count);
   }
 
+  /// Triggers multiple frames for `duration` amount of time or until there are
+  /// no longer any frames scheduled, whichever is earlier.
+  ///
+  /// This does not promise settle of the widget, and there maybe remaining
+  /// animation and therefore undisposed `Ticker`s.
+  ///
+  /// This will call [pump] at least once, even if no frames are scheduled when
+  /// the function is called, to flush any pending microtasks which may
+  /// themselves schedule a frame.
+  ///
+  /// `frequency` specify the frequency (in Hz) for frame requests. This is an
```

Comment:
This sentence should start with a capitalized word (typically, for parameters, we'd start with something like "The `frequence` parameter ...").

The sentence should also avoid using the term to define itself. I think `frequency` is a bit ambiguous here, and should probably be renamed something like `frameRefreshRate` or something like that.